lib/ckb/platform/ruby/key.rb

```diff
@@ -29,10 +29,10 @@ def initialize(privkey)
     def sign(data)
       privkey_bin = Utils.hex_to_bin(privkey)
       secp_key = Secp256k1::PrivateKey.new(privkey: privkey_bin)
-      signature_bin = secp_key.ecdsa_serialize(
-        secp_key.ecdsa_sign(Utils.hex_to_bin(data), raw: true)
+      signature_bin, recid = secp_key.ecdsa_recoverable_serialize(
```

Comment:
I suggest we add a new recoverable sign instead of changing the current `sign`. Just in case we need to use both in the future.

Reply:
updated